app/backend/resources/terms_of_service.md

```diff
@@ -1,17 +1,18 @@
 You must agree to these Terms to access the Couchers.org services and create an account.
 
-**Note: we'll update the Terms of Service as we approach the full launch and clarify the reasons and ways in which we may terminate or ban accounts as we develop these processes and solidify our moderation process.**
+*This is version 2.3 of the Terms of Service, last updated on 11th of June, 2024.*
 
-*This is version 2.2 of the Terms of Service, last updated on 20th of January, 2022.*
 
 # 1. Service provider
 
-Access to the Couchers.org platform ("platform", "service") is provided to you by the **Couchers, Inc.** ("Couchers.org", "we", "us"), a 501(c)(3) non-profit organization incorporated in the United States.
+Access to the Couchers.org platform ("platform", "service") is provided to you by **Couchers, Inc.** ("Couchers.org", "we", "us"), a 501(c)(3) non-profit organization incorporated in the United States.
 
 
 # 2. Eligibility
 
-You must be at least 18 years old to access the service.
+You must be at least 18 years old and a natural person to access the service.
+
+You are not permitted to have more than one account at a time on the service. If you are restricted from accessing the platform via a ban, suspension, or any other method, you may not create a new account. If you are in good standing, you may delete your account and sign up again later.
```

Comment:
```suggestion
Users are permitted to have only one account on Couchers.org. Creating, maintaining, accessing, or using multiple accounts is strictly prohibited under any circumstances. The only exception to this rule is if a user voluntarily deletes their account, they may return and create a new account. Users whose accounts have been suspended, banned, or restricted are not allowed to create new accounts. Violation of this policy will result in the termination of all associated accounts.

Accounts on Couchers.org are intended to foster genuine, platonic connections and community engagement within the context of travel and hospitality. Using accounts for purposes, including but not limited to data mining, surveillance, dating, sexual or romantic encounters, promotional endeavors, spamming, or any other activities that contradict the spirit of the platform or the community guidelines is strictly prohibited. Violations will lead to immediate account suspension or termination.
```

Reply:
I think the original is better: firstly using "You", since "user" is kind of the same as an account. So then we'd have to explain that user = you.

Also I think the "strictly prohibited..." wording is a bit too negative and draconian. I would prefer a more positive and neutral tone.

Reply:
sorted offline